index.html

```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
             </div>
         </div>
     </nav>
-    
+
+    <section class="top-hero-container">
+        <div class="hero-content">
+            <div>
+                <div>
+                    <h1 class="hero-heading">Build better processes</h1>
+                </div>
+                <div>
+                    <p class="hero-paragraph">Your customers won’t settle and neither should you. With Pluralsight,
+                        technology teams know more
+                        and
+                        work
+                        better together. Develop the skills you need for your top priorities. Improve workflows with
+                        actionable
+                        data. Build better—every time.
+                    </p>
+                </div>
+                <div class="hero-button-container">
+                    <div class="hero-button-size">
```

Comment:
Naming could be a bit better like hero-buttons. If the button sized is reused you can have like a large-button class...

Reply:
the above class just for display change to column.The button name is try-for-free-button and hope ok